### ISSUE TITLE
Model output for demo examples are blurry

### ISSUE THREAD

**Issue by zhanibekrysbek** [Kai-46/nerfplusplus#30]
After training with default configs listed in repo, I got much blurrier rendered images compared to what has been demonstrated. In particular truck scene has been trained for 500K iterations. Same situation for the train scene. I might be missing something important in the training phase.

What are the training params for the best model performance with high resolutions?

For reference see below two images, one is rendered second is the ground truth.

![image](https://user-images.githubusercontent.com/29447727/122595544-420aa900-d02e-11eb-9a7a-d19a912f650e.png)

![image](https://user-images.githubusercontent.com/29447727/122595561-48008a00-d02e-11eb-90b4-8d3bcaecac03.png)

**Comment by Kai-46** [comment]
Did you use the provided images and provided configuration parameters?

**Comment by zhanibekrysbek** [comment]
Yes. I used the default settings on the truck and train data set.

**Comment by Kai-46** [comment]
Okay. How many GPUs were used for parallel training then?

**Comment by zhanibekrysbek** [comment]
One V100 gpu with 32Gb. Model apparently uses up to 4Gb..

**Comment by Kai-46** [comment]
Oh, you might want to check [this line of code](https://github.com/Kai-46/nerfplusplus/blob/a30f1a5ad116e43aad90c426a966b2a3fcedaf7e/ddp_train_nerf.py#L345). I determine the number of rays to sample according to the available GPU memory at each training iteration; but I only take into account GPU with 12G or 24G mem. I used 4 12G-GPU during training, which means 2048 rays are sampled each iteration.

**Comment by zhanibekrysbek** [comment]
Got it! What param values (N_rand, chunk_size) you suggest for my settings for example?

**Comment by Kai-46** [comment]
Since your GPU has 32G mem, it might support a maximum N_rand=1300. chunk_size=10900. But if you only have access to this single GPU, it might be easier for you just to double the number of training iterations, as you are using half the number of rays I used.
